### PR TITLE
GetDefaultContainerPortNumber to GetDefaultContainerPortName

### PR DESCRIPTION
**Pull request by merlintang**
We need to get the port name of container for each job, so we would change the port number to port name in the job controller interface.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/common/41)

### CODE REVIEW

**Comment by merlintang** [comment]
@jian-he  can you review this ?

**Comment by jian-he** [comment]
LGTM

**Comment by merlintang** [comment]
@gaocegege can you look at this as well ?

File: job_controller/api/v1/interface.go [hunk 1]
```diff
@@ -62,7 +62,7 @@ type ControllerInterface interface {
 	GetDefaultContainerName() string
 
 	// Get the default container port number
```
Comment (terrytangyuan) [inline job_controller/api/v1/interface.go]: Update the comment here

**Comment by merlintang** [comment]
@gaocegege @terrytangyuan @jian-he  thanks for your comments.

**Comment by gaocegege** [comment]
Thanks for your contribution! :tada: :+1:

**Pull request merged**